R/bambu-processReads.R

```diff
@@ -87,8 +87,12 @@ bambu.processReadsByFile <- function(bam.file, genomeSequence, annotations,
         uniqueJunctions, runName = names(bam.file)[1],
         annotations, stranded, verbose)
     GenomeInfoDb::seqlevels(se) <- refSeqLevels
-    se = scoreReadClasses(se, genomeSequence, annotations, 
-        min.readCount = min.readCount, verbose = verbose)
+    se <- scoreReadClasses(se,genomeSequence, 
```

Comment:
please also call in the same order as they are defined in the function to avoid any errors if they are implictely called assuming a certain order

Reply:
done in 480cbb21bb36b8e95504515f6dab6157ea2520fc